flow-report/App.tsx

```diff
@@ -6,8 +6,10 @@
 
 import {FunctionComponent} from 'preact';
 import {useState} from 'preact/hooks';
+import type FlowResult from '../types/lhr/flow';
+import type LHResult from '../types/lhr/lhr';
```

Comment:
went with explicit type import because this is tsx. Could add `declare LH {}` to `flow-report.d.ts` instead if global types are preferred.

Reply:
Global types SGTM, I think it would also prevent a merge conflict with https://github.com/GoogleChrome/lighthouse/pull/12929

Reply:
> Global types SGTM

done